### PR TITLE
Add methods to get tags and tag folders

### PR DESCRIPTION
**Pull request by dawillcox**
A contribution, should you wish to accept: Adding a couple of methods to return tags and tag folders. That's essential to figure out some of the "details" of a people entry.

Sorry, no unit test. I don't really understand how those tests work, nor how to run them. I don't have a python development setup.

(Oh, and you might want to note somewhere that this requires python 3.)

### CODE REVIEW

File: breeze/breeze.py [hunk 1]
```diff
@@ -553,3 +555,70 @@ def list_pledges(self, campaign_id):
...
+            ]"""
+
+
+        params = []
+        if folder:
+            params.append('folder_id=%s' % folder)
+        return self._request('%s/?%s' % (ENDPOINTS.TAGS, '&'.join(params)))
+
+    def get_folders(api):
+        return api._request(ENDPOINTS.TAG_FOLDERS)
```
Comment (alexortizrosado) [inline breeze/breeze.py]: Thanks for adding the documentation! 

In this case, this line should appear _after_ the docblock, so moving it to after line 624 should do it.

File: breeze/breeze.py [hunk 2]
```diff
@@ -30,6 +30,8 @@
     CONTRIBUTIONS='/api/giving',
     FUNDS='/api/funds',
     PLEDGES='/api/pledges',
+    TAGS='/api/tags/list_tags/',
```
Comment (alexortizrosado) [inline breeze/breeze.py]: This ENDPOINTS enum is just meant to be the higher-level URL. In this case, would you mind removing  line 34

```
TAG_FOLDERS='/api/tags/list_folders/',
``` 

and then updating line 33 to be:

```
TAGS='/api/tags
```

File: breeze/breeze.py [hunk 3]
```diff
@@ -553,3 +555,70 @@ def list_pledges(self, campaign_id):
...
+            { ... }
+            ]"""
+
+
+        params = []
+        if folder:
+            params.append('folder_id=%s' % folder)
+        return self._request('%s/?%s' % (ENDPOINTS.TAGS, '&'.join(params)))
+
+    def get_folders(api):
```
Comment (alexortizrosado) [inline breeze/breeze.py]: Would you mind naming this `get_tag_folders` so its scope is more closely understood?

File: breeze/breeze.py [hunk 4]
```diff
@@ -553,3 +554,70 @@ def list_pledges(self, campaign_id):
...
+                "folder_id":"1539"
+            },
+            { ... }
+            ]"""
+
+
+        params = []
+        if folder:
+            params.append('folder_id=%s' % folder)
+        return self._request('%s/%s/?%s' % (ENDPOINTS.TAGS, 'list_tags', '&'.join(params)))
```
Comment (dawillcox) [inline breeze/breeze.py]: I question whether the '/' before '?' is needed here. Seems superfluous. But this matches existing usage.

**Comment by dawillcox** [comment]
I believe the request changes are complete.

**Comment by alexortizrosado** [comment]
@dawillcox - The changes look great!

**Pull request merged**